extensions/amp-bind/0.1/test/test-bind-impl.js

```diff
@@ -119,112 +116,142 @@ describes.realWin('amp-bind', {
     }).to.throw('Experiment "amp-bind" is disabled.');
   });
 
-  it('should scan for bindings when ampdoc is ready', () => {
+  it('should scan for bindings when ampdoc is ready', done => {
```

Comment:
Now that onBindReady() returns a promise, it looks like we need this to signal the tests are done after the promises resolve.